aws_lambda_builders/workflows/nodejs_npm_esbuild/workflow.py

```diff
@@ -126,10 +124,7 @@ def actions_with_bundler(
         ]
         esbuild_with_deps = EsbuildBundleAction(scratch_dir, artifacts_dir, bundler_config, osutils, subprocess_esbuild)
 
-        if osutils.file_exists(lockfile_path) or osutils.file_exists(shrinkwrap_path):
-            install_action = NodejsNpmCIAction(scratch_dir, subprocess_npm=subprocess_npm)
-        else:
-            install_action = NodejsNpmInstallAction(scratch_dir, subprocess_npm=subprocess_npm, is_production=False)
+        install_action = get_install_action(source_dir, scratch_dir, subprocess_npm, osutils, self.options)
```

Comment:
This one used to have `is_production` set to False. Did something change here or should this still be False?

Reply:
Yeah, we need the is_production flag to be different for npm and esbuild. Updated to reflect that.